### PR TITLE
account package: unit tests

### PR DESCRIPTION
**Pull request by rgeraldes**
This PR covers the status account package unit tests.

Edit1: still in progress. I can't add labels.
Edit2: RecoverAccount is not complete. There was an issue about it but I can't find it anymore.

### CODE REVIEW

**Comment by tiabc** [comment]
@rgeraldes can you resolve conflicts, please? I suppose, this pr is not in progress any more?

File: geth/account/accounts_test.go [hunk 1]
```diff
@@ -117,17 +407,394 @@ func (s *AccountsTestSuite) TestVerifyAccountPassword() {
...
 	address := gethcommon.HexToAddress(TestConfig.Account1.Address)
-	_, err = acctManager.VerifyAccountPassword(keyStoreDir, address.Hex(), TestConfig.Account1.Password)
-	s.NoError(err)
+	_, err = accountManager.VerifyAccountPassword(emptyKeyStoreDir, address.Hex(), TestConfig.Account1.Password)
+	require.NoError(err)
+}
+
+/*
+
+func (s *AccountsTestSuite) TestSelectAccount() {
```
Comment (influx6) [inline geth/account/accounts_test.go]: Are these tests to be removed?

Comment (rgeraldes) [reply geth/account/accounts_test.go]: still in progress: reviewing them one by one :)

**Comment by rgeraldes** [comment]
@tiabc I've finished this one. Regarding the conflicts it seems that the current test setting/architecture (e2e, unit, testing package removed) is not clear and this one should be merged only after some specifics have been decided. 
Please note that RecoverAccount is not ok functionality-wise. There was an issue covering this topic but I can't find it anymore.

**Comment by tiabc** [comment]
Thanks for the work @rgeraldes, we'll handle the rest!

**Comment by influx6** [comment]
Fix conflicts please.

**Comment by tiabc** [comment]
@b00ris please, handle this pr: pull the branch, close this pr, work on that for a while and push it as a new local pr.

**Comment by b00ris** [comment]
Got it.

**Pull request closed without merging**